go-controller/pkg/node/vrfmanager/vrf_manager.go

```diff
@@ -213,13 +219,21 @@ func (vrfm *Controller) sync(vrf vrf) error {
 			}
 		}
 	}
+	// Handover vrf routes into route manager to manage it.
+	for _, route := range vrf.routes {
+		err = vrfm.routeManager.AddRoute(route)
+		if err != nil {
+			klog.Errorf("Error adding route %s into VRF %s: %v", route.String(), vrf.name, err)
+		}
+	}
 
+	vrf.index = vrfLink.Attrs().Index
```

Comment:
no need to save the table index - you can get it from the link specifically from `netlink` type `vrf` that implements `netlink.Link`:
```
type Vrf struct {
	LinkAttrs
	Table uint32
}
```

Reply:
ah cool! thanks